optuna/samplers/_cmaes.py

```diff
@@ -177,6 +180,15 @@ def objective(trial):
                 versions without prior notice. See
                 https://github.com/optuna/optuna/releases/tag/v2.6.0.
 
+        with_margin:
+            If this is :obj:`True`, CMA-ES with margin is used.
+            Currently, this option cannot be used with ``use_separable_cma=True``.
```

Comment:
Also this option cannot be used with `source_trials`.

Reply:
I asked @nomuramasahir0 about this topic, and then he said "It's possible in theory. The thorough performance analysis has not been conducted though".

Therefore, seemingly, the use of the source_trials and with_margin options should not be prohibited on the interface. Let me resolve this comment :bow: